client/my-sites/subscribers/helpers/index.ts

```diff
@@ -1,11 +1,6 @@
 import { SubscriberListArgs } from '../types';
 
-const URL_PREFIX = 'https://wordpress.com';
-
-const getEarnPageUrl = ( siteSlug: string | null ) => `${ URL_PREFIX }/earn/${ siteSlug ?? '' }`;
-
-const getEarnPaymentsPageUrl = ( siteSlug: string | null ) =>
-	`${ URL_PREFIX }/earn/payments/${ siteSlug ?? '' }`;
+const getEarnPaymentsPageUrl = ( siteSlug: string | null ) => `/earn/payments/${ siteSlug ?? '' }`;
```

Comment:
Removed URL prefix, dynamic URLs for calypso pages work better with live branches, localhost, etc.

Reply:
...that said, need to review if this still works where the URL is used, in case those methods required full URL.